scripts/dashboard/install-e2e-test-deps.sh

```diff
@@ -6,6 +6,8 @@ if [[ ! $(arch) =~ (i386|x86_64|amd64) ]]; then
     exit
 fi
 
+sudo rm -f /etc/apt/sources.list.d/cortx-motr.list
```

Comment:
Are we adding this? If not, we shouldn't remove it. And that action should only be performed on a Debian/Ubuntu environment?